File: LEVEL1/두_정수_사이의_합_ComSenpai.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    
    //a가 b보다 더 큰 수인지를 검사
    if(a>b) { 
        int temp = a;
        a = b;
        b = temp;
    }
        
    //a부터 b까지 모두 더한다
    while(a<=b) {
        answer += a;
        a++;
    }
    return answer;
}

File: Programmers/LEVEL1/두_정수_사이의_합_ComSenpai.js
function solution(a, b) {
    var result = (a<b)?a:b; //a가 더 크다면 result == b

    while(a!=b){ //a와 b가 같아질 때 까지 반복한다
      result += (a<b)?++a:++b;
    }    

    return result;
}